suites/pacific/cephadm/tier-1_service_apply_spec.yaml

```diff
@@ -262,6 +269,22 @@ tests:
               args:              # sleep to get all services deployed
                 - sleep
                 - "120"
+  - test:
+      name: test_remove_service
+      desc: Perform removal of RBD mirroring service from the cluster
+      polarion-id: CEPH-83573761
+      module: test_remove_service.py
+      config:
+        service_name: rbd-mirror
+  - test:
+      name: test_redploy_service
+      desc: Verify redeploy for a specific service using ceph orch command
+      polarion-id: CEPH-83573728
+      module: test_redeploy_service.py
+      config:
+        service_name: mgr
+        image: docker.io/amk3798/cepg:testing
```

Comment:
Why private image ?

Reply:
Hi @vamahaja 

We are using same image provided with command in approved test case.

https://polarion.engineering.redhat.com/polarion/#/project/CEPH/workitem?id=CEPH-83573728